.tours/3---dynamic flows.tour

```diff
@@ -0,0 +1,95 @@
+{
+    "title": "3 - Dynamic Flows",
+    "steps": [
+        {
+            "file": "es-space-mgmt/main/default/aura/reservationHelperAura/reservationHelperAura.cmp",
+            "description": "The `reservationHelper` Aura component render flow interviews dynamically, by using the `lightning:flow` base component. You can read more about `lightning:flow` [here](https://developer.salesforce.com/docs/atlas.en-us.lightning.meta/lightning/components_using_flow_cmp.htm). We use Aura component embedding LWC component because LWC components do not provide ability to launch flows dynamically yet",
```

Comment:
The last sentence sounds grammatically off.

Reply:
Update the Aura component's name to `reservationHelperAura`... coz in the next step you talk about `reservationhelper` LWC.

Reply:
fixed!